protos/gen/go/v2/mailer/mailer.pb.go

```diff
@@ -0,0 +1,259 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.34.2
+// 	protoc        (unknown)
+// source: v2/mailer/mailer.proto
+
+package mailer
+
+import (
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type MailEvent struct {
```

Comment:
Судячи з задумки це скоріше команда, а не доменний івент
```suggestion
type MailCommand struct {
```

Reply:
сорі, не дуже зрозумів про що ти. Це якась концепція команди? Можеш будь ласка поділитися де я можу про це почитати?

Reply:
Так, https://blog.ttulka.com/events-vs-commands-in-ddd/
тут якраз є приклад про відправку емейлів

Reply:
так, це команда, я поправлю

Reply:
@fabl3ss тут доречі ще одну помилку помітив, я випадково сервіс скопіював у прото файл v2. Він тут зайвий, мені тільки message definition потрібно